src/librustc_resolve/diagnostics.rs

```diff
@@ -888,7 +911,7 @@ register_diagnostics! {
     E0418, // is not an enum variant, struct or const
     E0420, // is not an associated const
     E0421, // unresolved associated const
-    E0422, // does not name a structure
+//  E0422, // does not name a structure
```

Comment:
OK? Is that better? I noticed that some are commented out, while some are deleted.